vignettes/aj_write_man.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "10. Make Datasets Documentation with write_man()"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{10. Make Datasets Documentation with write_man()}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction
+
+R packages which contain datasets need documentation.  The Roxygen2 package helps write R manual pages but there are many details.  The `write_man()` function examines a dataset and writes a R file that contains Roxygen2 code which produces documents the dataset.  The documentation is written using Markdown to organize details including the number of rows and columns in the dataset, the names, the labels (if any) and types of variable.  The levels of categorical variables are also included.
```

Comment:
"The levels of categorical variables are also included" scares me. We need a check that there are far fewer than N levels. I don't know what "far fewer" means though, maybe 20% of the dataset? Maybe unique patient IDs, but then the function should have a check to not print thousands of lines of Roxygen code for all the levels. Maybe show the first 20 levels and then say "and XX more not shown".